### PR TITLE
Update blocklistConfig.json

### PR DESCRIPTION
**Pull request by O7zSO31IVg**
- Add curben pup-filter for blocking website hosted potential unwanted program

### CODE REVIEW

File: blocklistConfig.json [hunk 1]
```diff
@@ -1504,6 +1504,14 @@
       "url": "https://block.energized.pro/unified/formats/domains.txt",
       "pack": []
     },
+    {
+      "vname": "PUP filter", 
+      "format": "domains",
+      "group": "security",
+      "subg": "threat-intelligence-feeds"",
```
Comment (ignoramous) [inline blocklistConfig.json:1519]: extra quote?

**Comment by O7zSO31IVg** [comment]
- Add DuckDuckGo Tracker Radar for stronger tracker blocking

File: blocklistConfig.json [hunk 2]
```diff
@@ -1120,6 +1120,14 @@
       "url": "https://www.github.developerdan.com/hosts/lists/ads-and-tracking-extended.txt",
       "pack": []
     },
+    {
+      "vname": "DuckDuckGo Tracker Radar",
+      "format": "domains",
+      "group": "privacy",
+      "subg": "",
+      "url": "https://blokada.org/blocklists/ddgtrackerradar/standard/hosts.txt",
```
Comment (ignoramous) [inline blocklistConfig.json:1128]: Would you know the the license for Blokada's composition of DDG Tracker list? And if they're okay with RethinkDNS' use of it from their mirror?

Comment (O7zSO31IVg) [reply blocklistConfig.json:1128]: DuckDuckGo Tracker Radar database ruleset (and translated to domains syntax by blokada) is licensed under [CC-BY-NC-SA 4.0](https://github.com/duckduckgo/tracker-radar/raw/main/LICENSE)

Comment (ignoramous) [reply blocklistConfig.json:1128]: Unfortunately, a non-commercial license like `CC-BY-NC-SA`, isn't compatible with the FOSS license ServerlessDNS/RethinkDNS uses.

File: blocklistConfig.json [hunk 3]
```diff
@@ -1504,6 +1512,14 @@
...
       "pack": []
     },
+    {
+      "vname": "PUP filter", 
+      "format": "domains",
+      "group": "security",
+      "subg": "threat-intelligence-feeds",
+      "url": "https://curben.gitlab.io/malware-filter/pup-filter-domains.txt",
+      "pack": []
+    }
```
Comment (ignoramous) [inline blocklistConfig.json:1522]: missing a `comma` after `}`, like so: `},`

**Comment by O7zSO31IVg** [comment]
Add:
- curben URLHaus Blocklist
- curben phishing-filter
- The Block list project scam

**Comment by ignoramous** [comment]
If you can edit out the ddg list distributed by Blokada, I think we are good to merge this one in (provided we don't exceed size limits).

**Comment by O7zSO31IVg** [comment]
> If you can edit out the ddg list distributed by Blokada (which has licensing conflicts with RethinkDNS/ServerlessDNS), I think we are good to merge this one in (provided we don't exceed size limits).

So, what should I do

**Comment by ignoramous** [comment]
> do i need to remove ddgtrackerradar from the list

A bit too much work for you. I'll try to fix this commit (remove incompatible lists) and get it to merge (size limits are the only limiting factor)...

**Comment by ignoramous** [comment]
I intend to merge select lists from this PR by the next blocklist update (next week Thursday) once the size / limit tests are done. I'll report back here on the results.

**Comment by ignoramous** [comment]
See: https://github.com/serverless-dns/blocklists/issues/52#issuecomment-1264547211

**Pull request closed without merging**